imagepipeline/src/main/java/com/facebook/imagepipeline/core/ImagePipelineConfig.java

```diff
@@ -390,6 +396,11 @@ public Builder setCacheKeyFactory(CacheKeyFactory cacheKeyFactory) {
       return this;
     }
 
+    public Builder setHttpConnectionTimeout(int httpConnectionTimeout) {
```

Comment:
It would be better to include the unit at the end for all fields (e.g. `httpConnectionTimeoutMs`) so that you know what number you'd have to set.

Reply:
Added Ms for milliseconds as suggested.